saleor/order/migrations/0181_order_subtotal_as_a_field.py

```diff
@@ -0,0 +1,67 @@
+from django.db import migrations
+from django.db.models import F, Func, OuterRef, Subquery
+
+# One batch took 0.1s and 10.54MB at its peak
+BATCH_SIZE = 1000
+
+
+def update_order_subtotals(apps, schema_editor):
+    Order = apps.get_model("order", "Order")
+    OrderLine = apps.get_model("order", "OrderLine")
+
+    first_order = Order.objects.last()
+    if first_order is None:
+        return  # Exit if there are no orders
+    start_number = first_order.number
+
+    while True:
+        # right here we need to find only Orders which was not populated
+        # with subtotal in previous release, so the amount of rows to populate
+        # shouldn't be big
+        queryset = Order.objects.order_by("number").filter(
+            number__gte=start_number, subtotal_gross_amount__exact=0
```

Comment:
I've just realized that when there is an Order without any lines this method will be run indefinitely